core/src/main/java/de/danoeh/antennapod/core/storage/DBWriter.java

```diff
@@ -244,6 +244,19 @@ public static Future<?> clearPlaybackHistory() {
         });
     }
 
+    public static Future<?> deleteFromPlaybackHistory(FeedItem feedItem) {
+        return dbExec.submit(() -> {
+            PodDBAdapter adapter = PodDBAdapter.getInstance();
+            adapter.open();
+
+            feedItem.getMedia().setPlaybackCompletionDate(new Date(0));
+            adapter.setFeedMediaPlaybackCompletionDate(feedItem.getMedia());
+
+            adapter.close();
+            EventBus.getDefault().post(PlaybackHistoryEvent.listUpdated());
+        });
+    }
```

Comment:
Can also use the generic method:

```suggestion
    public static Future<?> deleteFromPlaybackHistory(FeedItem feedItem) {
        return addItemToPlaybackHistory(media, new Date(0));
    }
```